### PR TITLE
Blobless clone

### PR DESCRIPTION
**Pull request by cucxabong**

### CODE REVIEW

File: src/fast-forward.sh [hunk 1]
```diff
@@ -348,4 +348,6 @@ else
     echo "Not posting comment."
 fi
 
-exit $(cat $EXIT_CODE)
+# Always exit 0
+exit 0
```
Comment (nwalfield) [inline src/fast-forward.sh:354]: What's the rationale for this change?

File: src/fast-forward.sh [hunk 2]
```diff
@@ -180,8 +180,10 @@ LOG=$(mktemp)
         } | git credential approve
 
         CLONE_URL="${CLONE_URL%://*}://${GITHUB_ACTOR}@${CLONE_URL#*://}"
-        git clone --quiet --single-branch --filter=blob:none --branch "$BASE_REF" "$CLONE_URL" .
 
+        git clone --quiet --filter=blob:none --no-checkout "$CLONE_URL" .
+        git sparse-checkout set README.md
```
Comment (nwalfield) [inline src/fast-forward.sh:185]: Why restrict to `README.md`?  What if a repo doesn't have a `README.md` file?

**Review by nwalfield — COMMENTED** [review]
Thanks for this contribution!  I need to read some documentation before I can fully evaluate it though.

**Comment by cucxabong** [comment]
sorry, I open a wrong commit
